lib/src/main/java/com/jintin/clerk/lib/LogType.kt

```diff
@@ -0,0 +1,10 @@
+package com.jintin.clerk.lib
+
+enum class LogType(val color: String) {
```

Comment:
In my opinion `LogType` should not care what the color it is.
And maybe can add more attribute to distinguish type instead of  
using `ordinal` to get exact type because once we change the order it might broken.
What do you think?